x-pack/plugins/security_solution/server/lib/detection_engine/signals/threshold/get_threshold_bucket_filters.ts

```diff
@@ -34,7 +34,7 @@ export const getThresholdBucketFilters = async ({
 
       bucket.terms.forEach((term) => {
         if (term.field != null) {
-          (filter.bool.filter as ESFilter[]).push({
+          (filter.bool!.filter as ESFilter[]).push({
```

Comment:
cc @madirey

Reply:
This one is fine, as `filter.bool` is hard-coded directly above this. So we definitely won't see any change in behavior and this should work as expected.